test-data/unit/check-qualname.test

```diff
@@ -0,0 +1,10 @@
+[case testQualNameTypeCheck]
```

Comment:
There is no need to add a separate test file for this special case. Look where other tests for `__name__` etc. are defined and add your test cases there.